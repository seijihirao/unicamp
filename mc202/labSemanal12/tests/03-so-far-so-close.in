2
0 0 -1
1000000 100000000 -1
0 0
1000000 100000000
10000000 10000000
-1 -1
