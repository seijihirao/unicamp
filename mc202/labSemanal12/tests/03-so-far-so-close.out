(0,0) = -1
(1000000,100000000) = -1
(10000000,10000000) = 0
